blog/posts/managing-cookies.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Setting state using cookies
+date: 2020-10-14
+author: Giovanni Rago
+githubUser: ragog
+tags: 
+  - cookies
+---
+
+The [HyperText Transfer Protocol (HTTP)](https://developer.mozilla.org/en-US/docs/Web/HTTP#:~:text=Hypertext%20Transfer%20Protocol%20(HTTP)%20is,be%20used%20for%20other%20purposes.) is stateless, but cookies allow it to keep context consistent over the course of a session. In other words, by having our browser automatically exchange small amounts of data, we get to have websites recognise us and remember our preferences, the contents of our shopping baskets or the fact that we had just logged in to our account.
```

Comment:
Remember to replace `%20(HTTP)%20` for `%20%28HTTP%29%20` for the markdown issue